docs/releases/next.md

```
@@ -5,8 +5,8 @@
 
 ## What’s Changed
 
-*
+*  Better document moclojer's functions and workflow [PR #285](https://github.com/moclojer/moclojer/pull/285)
```

Markdownlint

Expected: 1; Actual: 2
Spaces after list markers

(MD030, list-marker-space)